docs/evaluate/temporal-cloud/security.mdx

```diff
@@ -65,6 +65,21 @@ You can find more details per cloud:
 
 * [AWS PrivateLink](aws-privatelink.mdx)
 
+### Temporal Nexus 
+
+Like Namespaces, a Nexus Endpoint is an account-scoped resource that is global within a Temporal Cloud account. 
+Any Developer role (or higher) in an account, who is also a Namespace Admin on the endpoint’s target Namespace, can manage (create/update/delete) a Nexus Endpoint. 
+All users with a Read-only role (or higher) in an account, can view and browse the full list of Endpoints.
+
+Runtime access from a Workflow in a caller Namespace to a Nexus Endpoint is controlled by an allowlist policy (of caller Namespaces) for each Endpoint in the Nexus API registry. 
+Workers authenticate with Temporal Cloud as they do today with mTLS certificates or API keys as allowed by the Namespace configuration. 
+Nexus requests are sent from the caller’s Namespace to the handler’s Namespace over a secure multi-region mTLS Envoy mesh.
+
+For payload encryption, the DataConverter works the same for a Nexus Operation as it does for other payloads sent between a Worker and Temporal Cloud. 
+There is support for a single DataConverter for inbound inputs and outbound outputs, which means the encryption key often used in a custom DataConverter must be the same for both the caller and the handler. 
```

Comment:
```suggestion
There is support for a single DataConverter for inbound inputs and outbound outputs, which means the encryption key often used in a custom DataConverter must be available in both the caller and the handler.
This may be achieved by using the same encryption key in both the caller and handler, or passing a KMS encryption key ID via payload metadata such that it can be use to obtain the encryption key to decrypt a payload.
```

Reply:
updated